pytest/tests/loadtest/locust/download_contracts.sh

```diff
@@ -2,8 +2,7 @@
 #
 # Downloads the WASM contracts necessary for all workloads and stores them in "res" folder.
 
-cd res
-wget https://raw.githubusercontent.com/NearSocial/social-db/master/res/social_db_release.wasm -O social_db.wasm
-wget https://raw.githubusercontent.com/sweatco/sweat-near/main/res/sweat.wasm -O sweat.wasm
-ln -s ../../../../../runtime/near-test-contracts/res/fungible_token.wasm fungible_token.wasm
-ln -s ../../../../../runtime/near-test-contracts/res/backwards_compatible_rs_contract.wasm congestion.wasm
+wget https://raw.githubusercontent.com/NearSocial/social-db/master/res/social_db_release.wasm -O res/social_db.wasm
+wget https://raw.githubusercontent.com/sweatco/sweat-near/main/res/sweat.wasm -O res/sweat.wasm
+cp ../../../../runtime/near-test-contracts/res/fungible_token.wasm res/fungible_token.wasm
+cp ../../../../runtime/near-test-contracts/res/backwards_compatible_rs_contract.wasm res/congestion.wasm
```

Comment:
Why copy instead of links? 

nit: It would be nicer to put the `../../../../runtime/near-test-contracts/res/` in a variable.

Reply:
I will move the script directory to a variable (also following the other suggestion).

I do not understand your question regarding the links?

Reply:
Reverted back to links now (I think CI failures are related, re-trying). In fact if the copying works, the previous version with links (from the same source) should have worked.